src/plugins/intel_cpu/src/emitters/x64/jit_dnnl_ext_emitters.hpp

```diff
@@ -22,6 +22,10 @@ class jit_relu_emitter : public jit_dnnl_emitter {
 
             set_injector();
         }
+    void print_debug_info() const override {
+        std::cerr << "Emitter type name:" << get_type_name(this) << "\n";
+        std::cerr << "dnnl_alg_kind:" << kind << " alpha" << alpha << " beta" << beta << "\n";
+    }
```

Comment:
Only by calling the override version from derived class, we know more details which derived emitters the error is from(get the derived emitters name). This could give more specific info.

Reply:
Actually we can get the derived emitters name even without overriding the `print_debug_info()` method since we pass to `get_type_name` a `jit_emitter` pointer which points to an object of a derived class. E.g. I removed `print_debug_info()` for `jit_load_emitter` and get the following output:
> Emitter type name:**ov::intel_cpu::jit_load_emitter** This is default info from base jit_emitter. Exact emitter type name is not set.

So my suggestion is to leave overridden `print_debug_info()` methods only for the classes which print any additional information except emitter type name

Reply:
yes, you are right. I previously reuse base version of print_debug_info() in jit_emitter on a unfinished basis, so not get right name. Have done, thanks Vladislav!